app/views/bot_instances/index.html.erb

```diff
@@ -17,4 +17,8 @@
       <% end %>
     </li><br/>
   <% end %>
-</ul>
\ No newline at end of file
+</ul>
+
+<% if user_signed_in? && (current_user.has_role?(:owner, @bot) || current_user.has_role?(:collaborator, @bot))%>
```

Comment:
I *think* Rolify has a `has_any_role?` method that can query for multiple roles at once.